.github/workflows/chart_publish_insiders.yml

```diff
@@ -0,0 +1,43 @@
+name: Release Charts (Insiders)
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.8.0
+      
+      - name: Get metadata
+        id: metadata
+        uses: contiamo/git-metadata-action@main
+
+      - run: |
+          sed -i 's/appVersion:.*/appVersion: insiders/g' charts/*/Chart.yaml
+          sed -i '/^version:/ s/$/-insiders.${{ steps.metadata.outputs.shortSHA }}/' charts/*/Chart.yaml
```

Comment:
this will always update all charts regardless if there is any change, do we want something similar to cr-action? only publish charts that have been changed.